new/change/spec/write-package-changelogs.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { envWithConfig } from '../../fixtures/env';
+import type { Module } from '../../types';
+// mock logger
+const mockLog = jest.fn();
+const mockError = jest.fn();
+const mockLogger = jest.fn(() => ({ log: mockLog, error: mockError }));
+jest.doMock('../../helpers/logger', () => ({ logger: mockLogger }));
+
+describe('writePackageChangelogs Module', () => {
+  let writePackageChangelogs: Module;
+
+  beforeAll(() => {
+    writePackageChangelogs =
+      require('../write-package-changelogs').writePackageChangelogs;
```

Comment:
why do this vs just a simple import? you don't seem to be mocking anything 🤔

Reply:
`logger` is being mocked (line 7)